slice/Glacier2/PermissionsVerifier.ice

```diff
@@ -5,52 +5,50 @@
 [["cpp:dll-export:GLACIER2_API"]]
 [["cpp:doxygen:include:Glacier2/Glacier2.h"]]
 [["cpp:header-ext:h"]]
+
 [["cpp:include:Glacier2/Config.h"]]
 
+[["java:package:com.zeroc"]]
 [["js:module:@zeroc/ice"]]
-
 [["python:pkgdir:Glacier2"]]
 
 #include "SSLInfo.ice"
 
-[["java:package:com.zeroc"]]
-
 module Glacier2
 {
-    /// This exception is raised if a client is denied the ability to create a session with the router.
+    /// The exception that is thrown when a client is not allowed to create a session.
```

Comment:
"The exception that is thrown when ..." is the standard C# description for exceptions.